src/casp/services/api/schemas/cell_operations.py

```diff
@@ -3,7 +3,7 @@
 from pydantic import BaseModel, Field
 
 
-class MatchInfo(BaseModel):
+class NeighborhoodSummaryCellTypeStatistics(BaseModel):
```

Comment:
`NeighborhoodSummaryCellTypeStatistics` -> ``NeighborhoodCellTypeSummaryStatistics`. "Summary Statistics" is a mathematical term, e.g. median, p25, etc are all "summary statistics".